src/muya/lib/ui/imageToolbar/index.js

```diff
@@ -53,6 +56,10 @@ class ImageToolbar extends BaseFloat {
     const { icons, oldVnode, toolbarContainer, imageInfo } = this
     const { attrs } = imageInfo.token
     const dataAlign = attrs['data-align']
+    let imageIsLocal = true
+    if (URL_REG.test(imageInfo.token.src) || URL_REG.test(attrs.src)) {
+      imageIsLocal = false
+    }
```

Comment:
Could we outsource this into a function with `imageInfo` as parameter because its used multiple times?